sympy/solvers/ode/single.py

```diff
@@ -590,7 +590,7 @@ class FirstLinear(SinglePatternODESolver):
     .. math:: dy/dx + P(x) y = Q(x)\text{.}
 
     These kinds of differential equations can be solved in a general way.  The
-    integrating factor `e^{\int P(x) \,dx}` will turn the equation into a
+    integrating factor `e^{\int P(x) \, dx}` will turn the equation into a
```

Comment:
The `\,` in latex is different from other commas since it represents a small space. I don't think that we need to also add a space after this.